gax/operations_stub.h

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Google Inc.  All rights reserved
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GAPIC_GENERATOR_CPP_GAX_OPERATIONS_STUB_H_
+#define GAPIC_GENERATOR_CPP_GAX_OPERATIONS_STUB_H_
+
+#include "google/longrunning/operations.pb.h"
+
+#include "call_context.h"
```

Comment:
This could fail in interesting ways if the application has a `call_context.h` file in their local directory.... Consider always using the full path for your includes, even when including a local file.

Reply:
Oy. Good point.

Reply:
Tracked by #36